internal/pkg/cli/dialog/use_template.go

```diff
@@ -93,9 +93,9 @@ func (d *useTmplDialog) askInputs(stepsGroups input.StepsGroupsExt) error {
 			}
 		}
 
-		// Use default or empty value if we don't ask for the input
+		// Use empty value if we don't ask for the input
 		if !available {
-			return d.addInputValue(inputDef.DefaultOrEmpty(), inputDef, false)
+			return d.addInputValue(inputDef.Empty(), inputDef, false)
```

Comment:
Ak nejaky vstup nezobrazujeme, tak do sablony dame prazdnu hodnotu, ... je to tak viac logicke.
Rovnako je to v API https://github.com/keboola/keboola-as-code/blob/e5239f1eb43d4f367357a7bee0acec7a75a4bd85/internal/pkg/api/server/templates/service/inputs.go#L56